src/mono/wasm/debugger/BrowserDebugProxy/MemberObjectsExplorer.cs

```diff
@@ -282,6 +287,8 @@ async Task<JArray> GetRootHiddenChildrenForProperty()
         {
             if (state is DebuggerBrowsableState.RootHidden)
             {
+                if (namePrefix == "valueTypeEnumRootHidden")
+                    Console.WriteLine($"namePrefix = {namePrefix}; typeName = {typeName}");
```

Comment:
Can we drop this?

Reply:
It's a debugging leftover that I overlooked, thanks.